docs/concepts/scheduler.md

```diff
@@ -69,6 +69,13 @@ It filters out `Seed`s
 
 After this filtering process the least utilized seed, i.e., the one with the least number of shoot control planes, will be the winner and written to the `.spec.seedName` field of the `Shoot`.
 
+## `seedSelector` field in the `Shoot` specification
```

Comment:
Can you update Line 51?
`Subsequently the Gardener Controller Manager picks up and starts to create the cluster on the specified seed.` 
`Gardener Controller Manager` -> `Gardenlet`

Reply:
Also there is a missing closing bracket in line 55.